lib/is-prerelease.js

```diff
@@ -1,3 +1,9 @@
-export default function isPrerelease({ type, main }) {
-  return type === "prerelease" || (type === "release" && !main);
+export default function isPrerelease({type, main, prerelease}) {
+    if (typeof prerelease == 'boolean' && prerelease === false) {
+        return false;
+    }
+    return type === "prerelease"
+        || (type === "release" && !main)
+        || typeof prerelease == 'string'
+        || prerelease === true
```

Comment:
```suggestion
    if (typeof prerelease === "boolean" && prerelease === false) {
        return false;
    }
    return type === "prerelease"
        || (type === "release" && !main)
        || typeof prerelease == "string"
        || prerelease === true
```

I don't think `prerelease` is ever `true`, its either `false` or `[git_ref]` 😉

Reply:
The problem here is that prerelease can either be `true` or `false` / `undefined` or a string value as of the definition I read in semantic release. That's the reason why I don't want to consider to only return prerelease if it is undefined but only if it is `prerelease === true`. If prerelease is undefined it would return false - right?

Reply:
Good thought, It's okay for addressing the Edge case... We can always iterate over it later 😉